app/committees/controllers.py

```diff
@@ -153,18 +155,22 @@ def create_committee(user, user_data):
                 if "committee_img" in user_data:
                     com_img = base64.b64decode(user_data["committee_img"])
                     new_committee.committee_img = com_img
-
+                
+                print(new_committee)
+                print("TESTING new committee")
```

Comment:
Print statements, while helpful during development, should be removed before merging into the master branch.